keps/sig-auth/2579-psp-replacement/README.md

```diff
@@ -609,39 +609,23 @@ pod_security_evaluations_total
 
 The metric will use the following labels:
```

Comment:
cc @dashpole (listed in the KEP as the instrumentation reviewer) and @logicalhan (who had questions about cardinality in the [implementing PR](https://github.com/kubernetes/kubernetes/pull/104217#discussion_r687912865))

Reply:
lgtm